parachain/src/service.rs

```diff
@@ -731,15 +734,24 @@ where
         // Channel for the rpc handler to communicate with the authorship task.
         let (command_sink, commands_stream) = futures::channel::mpsc::channel(1024);
 
-        let pool = transaction_pool.pool().clone();
-        let import_stream = pool.validated_pool().import_notification_stream().map(|_| {
-            sc_consensus_manual_seal::rpc::EngineCommand::SealNewBlock {
-                create_empty: true,
-                finalize: true,
-                parent_hash: None,
-                sender: None,
+        let commands_stream: Box<dyn futures::Stream<Item = EngineCommand<H256>> + Send + Sync + Unpin> = match sealing
+        {
+            Sealing::Instant => {
+                let import_stream = transaction_pool
+                    .pool()
+                    .validated_pool()
+                    .import_notification_stream()
+                    .map(|_| EngineCommand::SealNewBlock {
+                        create_empty: true,
+                        finalize: true,
+                        parent_hash: None,
+                        sender: None,
+                    });
+                // combine so we can still seal blocks manually
+                Box::new(futures::stream_select!(commands_stream, import_stream))
```

Comment:
Just noting that other projects (such as [moonbeam](https://github.com/moonbeam-foundation/moonbeam/blob/4670c614274645139c7ba4d37fa58a1a4e7f2ec3/node/service/src/lib.rs#L1025)) don't allow manual-seal as well as instant-seal.